docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md

```diff
@@ -53,7 +53,7 @@ The main difference between [CTCGreedyDecoder](CTCGreedyDecoder_1.md) and CTCGre
 
 * **2**: `sequence_length` - input tensor of type *T_I* of shape `[N]` with sequence lengths. The values of sequence length must be less or equal to `T`. **Required.**
 
-* **3**: `blank_index` - scalar or 1D tensor with 1 element of type *T_I*. Specifies the class index to use for the blank class. The `blank_index` is not saved to the result sequence and it is used for post-processing. Default value is `C-1`. **Optional**.
+* **3**: `blank_index` - scalar or 1D tensor with 1 element of type *T_I*. Specifies the class index to use for the blank class. Regardless of the value of `merge_repeated` attribute, if the maximum index of a given `T` and `N` corresponds to the `blank_index`, no new element is emitted. Default value is `C-1`. **Optional**.
```

Comment:
Please correct as so:
if the output index for a given batch and time step corresponds to the `blank_index`, ...

maximum index also sounds incorrect because algorithm selects an index with maximum probability.